http/restore_service.go

```diff
@@ -80,8 +81,24 @@ func (h *RestoreHandler) handleRestoreKVStore(w http.ResponseWriter, r *http.Req
 	defer span.Finish()
 
 	ctx := r.Context()
+	defer r.Body.Close()
```

Comment:
I'm curious as to why we need to explicitly close the request body? I thought the server handled that automatically.

Reply:
Ah I didn't realize it was automatic, I'll revert these lines